code/game/area/areas/mining.dm

```diff
@@ -4,6 +4,7 @@
 	icon_state = "mining"
 	has_gravity = STANDARD_GRAVITY
 	area_flags = VALID_TERRITORY | UNIQUE_AREA | FLORA_ALLOWED | CULT_PERMITTED
+	ambient_buzz = 'sound/ambience/magma.ogg'
```

Comment:
`sound/ambience/magma.ogg` seems to be making a small clicking sound whenever it loops. I *think* this is because the actual sound file doesn't have any fade-in, but I'm not sure. (It might also just be something on my end)

Reply:
I made some adjustments to alleviate this. It's due to some weird audio compression or bass or something. I'm no audio nerd, they'd probably know.